src/plugins/imagery/components/ImageryViewLayout.vue

```diff
@@ -185,15 +186,17 @@ export default {
         setSelectedImage(image) {
             // If we are paused and the current image IS selected, unpause
             // Otherwise, set current image and pause
-            if (this.isPaused && image.selected) {
-                this.paused(false);
-                this.unselectAllImages();
-            } else {
-                this.imageUrl = this.getImageUrl(image);
-                this.time = this.getTime(image);
-                this.paused(true);
-                this.unselectAllImages();
-                image.selected = true;
+            if(image) {
```

Comment:
space after if  `if (`
Also can flatten this using (avoid nested if else) 😃 
```
if (!image) {
    return;
}
```

Reply:
Done and Done!